schema/github.schema.json

```
@@ -132,13 +134,26 @@
             "description": "Regular expression which matches against the name of a GitHub repository (\"owner/name\").",
             "type": "string",
             "format": "regex"
+          },
+          "size": {
+            "description": "If set, repositories with a size above the specified one will be excluded. Specify in kb.",
```

That seems like a leftover now that we parse units? 
```suggestion
            "description": "If set, repositories with a size above the specified one will be excluded.",
```